kubejs/server_scripts/base/recipes/minecraft/workbench.js

```diff
@@ -0,0 +1,3 @@
+onEvent("recipes", event => {
+    event.shapeless('1x minecraft:crafting_table', ['#forge:workbench']);
```

Comment:
The `1x` is redundant here, as that's the default.

I'd like this recipe/file, but moved to the location `kubejs\server_scripts\base\recipes\minecraft\shapeless.js`